shell/packages/sandstorm-ui-app-details/app-details-common.js

```diff
@@ -0,0 +1,14 @@
+SandstormAppDetails = function(db, quotaEnforcer, appId) {
```

Comment:
This object is only constructed on the client, right? Why have a "-common.js" at all?